flocker/node/script.py

```diff
@@ -294,7 +365,7 @@ class AgentServiceFactory(PClass):
     # This should have an explicit interface:
     # https://clusterhq.atlassian.net/browse/FLOC-1929
     deployer_factory = field(mandatory=True)
-    get_external_ip = field(initial=_get_external_ip, mandatory=True)
```

Comment:
:question: These all belong in #2725 where @moshez switches us to a vendored Pyrsistent, I think. Or alternatively we can create an "upgrade-pyristent" branch and *then* switch to using our vendored repo once we've merged all the optimizations there.